kubesae/ansible/deploy.py

```diff
@@ -14,13 +14,26 @@ def install_requirements(c):
         c.run(f"ansible-galaxy install -f -r '{req_file}' -p roles/")
 
 
-@invoke.task(pre=[install_requirements], default=True)
-def ansible_deploy(c, env=None, tag=None):
+def get_verbosity_flag(verbosity):
+    """
+    Given an integer, return a string with that number of `v` characters, prefixed
+    by a hyphen, for use as a flag to the ansible-playbook command. If verbosity is
+    zero, return the empty string.
+    """
+    v_flag = ""
+    if verbosity:
+        v_flag = "-{'v'*verbosity}"
+    return v_flag
+
+
+@invoke.task(default=True)
+def ansible_deploy(c, env=None, tag=None, verbosity=1):
```

Comment:
I've gotten used to adding a lot of v's, but would a default of 0 be preferable? Not really sure, just a thought.

Reply:
Yeah, I waffled between those two options. In the end, I thought that it would be preferable for devs to not have to add v's by default, but for the CI command to have a `--verbosity=0` flag, since it's only a one time configuration in the CI config file.

Reply:
I'll go ahead and merge this now, but I would be 100% open to changing the defaults.